lib/src/judo_tab.dart

```diff
@@ -5,6 +5,10 @@ class JudoTab extends StatefulWidget {
     Key key,
     @required this.col,
     @required this.row,
+    this.disabledTabs, // TODO: JNG-2874
+    this.hiddenTabs,
```

Comment:
Optional parameters *MUST* have default values. This helps us by negating the need of internal null checks!

Reply:
Never mind, language has difficulties working as expected.